### PR TITLE
Prepare release.

### PR DESCRIPTION
**Pull request by twittner**
Increment versions and update CHANGELOGs plus some dependencies.

### CODE REVIEW

File: muxers/yamux/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# 0.19.1 [2020-06-19]
+
+Deprecated method `Yamux::is_remote_acknowledged` has been removed
+as part of [PR 1616](https://github.com/libp2p/rust-libp2p/pull/1616).
+
```
Comment (mxinden) [inline muxers/yamux/CHANGELOG.md:5]:
```suggestion
```

File: protocols/noise/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# 0.19.1 [2020-06-19]
+
+- Re-add noise upgrades for IK and IX
+  ([PR 1580](https://github.com/libp2p/rust-libp2p/pull/1580)).
+
+- Updated dependencies.
+
```
Comment (mxinden) [inline protocols/noise/CHANGELOG.md:7]:
```suggestion
```

File: transports/websocket/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# 0.20.0 [2020-06-19]
+
+- Updated `soketto` dependency which caused some smaller
+  API changes ([PR 1603](https://github.com/libp2p/rust-libp2p/pull/1603)).
+
```
Comment (mxinden) [inline transports/websocket/CHANGELOG.md:5]:
```suggestion
```

**Comment by romanb** [comment]
Interestingly another test now stalls:

> test length_delimited::tests::writing_reading ... test length_delimited::tests::writing_reading has been running for over 60 seconds

with any `1.6.x` version of `async-std` (tested by @twittner). The noise smoke tests work now, so this may be a separate issue from the one that was resolved in `async-std-1.6.2`.

**Comment by tomaka** [comment]
> I don't have an opinion in regards to the release process change (e.g. one changelog per crate), thus I will leave it up to Roman and Pierre to approve.

I'm opposed to this change, but I'm also glad to not have the responsibility for the releases myself anymore.

**Comment by romanb** [comment]
> > I don't have an opinion in regards to the release process change (e.g. one changelog per crate), thus I will leave it up to Roman and Pierre to approve.
> 
> As discussed on Riot I'm opposed to this change, but I'm also glad to not have the responsibility for the releases myself anymore.

Just for clarity, I understood that you agreed to the change of process as long as you don't have to make the releases. Now it seems you won't approve any further releases because you disagree with the process (that is, multiple independent crates) despite the fact that you don't have to prepare the releases any more? :expressionless:

**Comment by tomaka** [comment]
> Now it seems you won't approve any further releases because you disagree with the process

Not agreeing doesn't mean I disagree.

My intention is that you should feel free to go ahead and handle the release process without me.


The only reason I commented on this PR is to respond to this comment:

> thus I will leave it up to Roman and Pierre to approve.

**Comment by romanb** [comment]
> Not agreeing doesn't mean I don't want this to be merged.

Wanting a PR to be merged is what it means to approve it.

> My intention is that you should feel free to go ahead and handle the release process without me.

If you don't approve releases despite wanting them merged, the result is that no release gets done as soon as just one of @twittner  or myself is absent, despite you wanting it merged. If that makes sense to you, well :man_shrugging:

**Comment by romanb** [comment]
For the record, the latest release(s) of `smol` underlying `async-std-1.6` seemed to have helped with the stalling test `length_delimited::tests::writing_reading`. May have been https://github.com/stjepang/smol/pull/170.

**Pull request merged**